dist/jquery.jcarousel-autoscroll.js

```diff
@@ -78,7 +78,7 @@
                 .one('jcarousel:animateend', this.onAnimateEnd);
 
             this._timer = setTimeout($.proxy(function() {
-                this.carousel().jcarousel('scroll', this.options('target'));
+                this.carousel().jcarousel(this.options('method'), this.options('target'));
```

Comment:
you forgot default option, and please add to documentation about it

Reply:
Done